### ISSUE TITLE
copyMessage JSON output not on a single line

### ISSUE THREAD

**Issue by moorereason** [minio/mc#3029]
## Expected behavior

One JSON object per line.

## Actual behavior

`copyMessage` object [is printed](https://github.com/minio/mc/blob/RELEASE.2019-12-24T23-41-36Z/cmd/cp-main.go#L203-L209) on multiple lines.

## Steps to reproduce the behavior

```
$ mc --json cp play/test/any_filename . | tee /dev/null
{
 "status": "success",
 "source": "play/test/any_filename",
 "target": "any_filename",
 "size": 1098,
 "totalCount": 0,
 "totalSize": 0
}
{"status":"success","total":0,"transferred":1098,"speed":1239.9256198658913}
```
## mc --version

mc version RELEASE.2019-12-24T23-41-36Z

## System information

`mc` tested on windows/amd64 (Windows 10) and linux/amd64 (archlinux).

**Comment by harshavardhana** [comment]
If you pipe it to a script it will be on a single line @moorereason for interactive reasons it's printed as JSON lines which is a valid JSON document. 

```
~ mc ls --json | more
```

**Comment by moorereason** [comment]
@harshavardhana, if you will take a closer look my report, I do pipe the output to `tee`.

Pipe it to `more` or `less`.  Redirect it to a file.  Run `mc` within Powershell to capture the output, which is how I discovered the issue.  I can reproduce it every time.

Please reopen the issue.

**Comment by harshavardhana** [comment]
Yes shall be fixed in #3030
